--- a/net/minecraft/network/protocol/configuration/ClientboundUpdateEnabledFeaturesPacket.java
+++ b/net/minecraft/network/protocol/configuration/ClientboundUpdateEnabledFeaturesPacket.java
@@ -12,9 +12,9 @@
     public static final StreamCodec<FriendlyByteBuf, ClientboundUpdateEnabledFeaturesPacket> STREAM_CODEC = Packet.codec(
         ClientboundUpdateEnabledFeaturesPacket::write, ClientboundUpdateEnabledFeaturesPacket::new
     );
 
     private ClientboundUpdateEnabledFeaturesPacket(FriendlyByteBuf buffer) {
-        this(buffer.readCollection(HashSet::new, FriendlyByteBuf::readResourceLocation));
+        this((HashSet<ResourceLocation>)buffer.readCollection(HashSet::new, FriendlyByteBuf::readResourceLocation));
     }
 
     private void write(FriendlyByteBuf buffer) {
